utils/shipping-actions.js

```diff
@@ -1,5 +1,11 @@
 import { ACCOUNT_NAMES } from './constants'
 
+export function selectCountry(country) {
+  cy.wait(3000)
```

Comment:
Is it breaking without the wait? If yes, maybe we can increase the maximum waiting time so it doesn't crash bellow 3s

Reply:
it didn't work without a waiting time? you want me to increase this time?

Reply:
I would like to remove it.

If `cy.get` bellow is timing out, maybe we can refactor into something like what is suggested here: https://twitter.com/housecor/status/1395508128960155649